ydb/core/tx/datashard/datashard_user_table.h

```diff
@@ -273,10 +274,20 @@ struct TUserTable : public TThrRefBase {
                 nameToId.emplace(column.Name, id);
             }
 
-            auto fillColumnIds = [&nameToId](const auto& columnNames, TVector<ui32>& columnIds) {
+            auto fillColumnIds = [&nameToId, type = Type](const auto& columnNames, TVector<ui32>& columnIds) {
                 columnIds.reserve(columnNames.size());
                 for (const auto& columnName : columnNames) {
                     auto it = nameToId.find(columnName);
+
+                    if (it == nameToId.end() && type == NKikimrSchemeOp::EIndexType::EIndexTypeGlobalVectorKmeansTree) {
+                        Y_ABORT_UNLESS(columnName == NTableIndex::NTableVectorIndex::IdColumn || 
+                                       columnName == NTableIndex::NTableVectorIndex::LevelColumn ||
+                                       columnName == NTableIndex::NTableVectorIndex::CentroidColumn ||
+                                       columnName == NTableIndex::NTableVectorIndex::IdsColumn
+                                       );
+                        continue;
+                    }
+
```

Comment:
This is incorrect, because `TTableIndex` isn't `implIndexTable`, it's kind of index definition where `nameToId` is from **indexed** table

In case of vector index
```
KeyColumnIds: id of embedding
DataColumnIds: id of data columns
```

But truly it doesn't matter because filling this vector needed only for async indexes now

Reply:
`TTableIndex`  is `implIndexTable`. Its name is `idx_vector`.  It's created [here](https://github.com/ydb-platform/ydb/blob/b0efa0f277ab640a2abfcec50bae4578969fe6a1/ydb/core/tx/datashard/datashard_user_table.cpp#L319).
 
 But I have double checked. This code is redundant now. It's from previous versions of vector index. I have reverted it.